GVRf/Framework/src/org/util/jassimp/AiConfigOptions.java

```diff
@@ -0,0 +1,745 @@
+    // * <br>
+    // * Property type: String. Default value: guessed.
+    // */
+    // #define AI_CONFIG_IMPORT_OGRE_MATERIAL_FILE "IMPORT_OGRE_MATERIAL_FILE"
+    //
+    //
+    // //
+    // ---------------------------------------------------------------------------
+    // /** @brief Ogre Importer detect the texture usage from its filename
+    // *
+    // * Normally, a texture is loaded as a colormap, if no target is specified
+    // in the
+    // * materialfile. Is this switch is enabled, texture names ending with _n,
+    // _l, _s
+    // * are used as normalmaps, lightmaps or specularmaps.
+    // * <br>
+    // * Property type: Bool. Default value: false.
+    // */
+    // #define AI_CONFIG_IMPORT_OGRE_TEXTURETYPE_FROM_FILENAME
+    // "IMPORT_OGRE_TEXTURETYPE_FROM_FILENAME"
+    //
+    //
+    //
+    // //
+    // ---------------------------------------------------------------------------
+    // /** @brief Specifies whether the IFC loader skips over IfcSpace elements.
+    // *
+    // * IfcSpace elements (and their geometric representations) are used to
+    // * represent, well, free space in a building storey.<br>
+    // * Property type: Bool. Default value: true.
+    // */
+    // #define AI_CONFIG_IMPORT_IFC_SKIP_SPACE_REPRESENTATIONS
+    // "IMPORT_IFC_SKIP_SPACE_REPRESENTATIONS"
+    //
+    //
+    // //
+    // ---------------------------------------------------------------------------
+    // /** @brief Specifies whether the IFC loader skips over
+    // * shape representations of type 'Curve2D'.
+    // *
+    // * A lot of files contain both a faceted mesh representation and a outline
+    // * with a presentation type of 'Curve2D'. Currently Assimp doesn't convert
+    // those,
+    // * so turning this option off just clutters the log with errors.<br>
+    // * Property type: Bool. Default value: true.
+    // */
+    // #define AI_CONFIG_IMPORT_IFC_SKIP_CURVE_REPRESENTATIONS
+    // "IMPORT_IFC_SKIP_CURVE_REPRESENTATIONS"
+    //
+    // //
+    // ---------------------------------------------------------------------------
+    // /** @brief Specifies whether the IFC loader will use its own, custom
+    // triangulation
+    // * algorithm to triangulate wall and floor meshes.
+    // *
+    // * If this property is set to false, walls will be either triangulated by
+    // * #aiProcess_Triangulate or will be passed through as huge polygons with
+    // * faked holes (i.e. holes that are connected with the outer boundary
+    // using
+    // * a dummy edge). It is highly recommended to set this property to true
+    // * if you want triangulated data because #aiProcess_Triangulate is known
+    // to
+    // * have problems with the kind of polygons that the IFC loader spits out
+    // for
+    // * complicated meshes.
+    // * Property type: Bool. Default value: true.
+    // */
+    // #define AI_CONFIG_IMPORT_IFC_CUSTOM_TRIANGULATION
+    // "IMPORT_IFC_CUSTOM_TRIANGULATION"
+    //
```

Comment:
please don't check-in commented-out code.  especially a large volume of commented-out code.